### PR TITLE
Added Key Translation & Mobile App

### PR DESCRIPTION
**Pull request by AymenRachid**
## Description

`Translation of some keys added`
`New upcoming mobile app added`
`Focusing first on the temperature feature`

## Related Issue

`Some of the keys not translated yet`
`Improvement for the V3.0 reef-pi, including and supporting a new mobile app`

## Motivation and Context

`As a reef enthusiast, this project is interesting for all the community. I would like to help creating a mobile app since it will be easier to control your reef directly via mobile instead of having to use the computer.
This is a small contribution to begin with and better enhancement will come hopefully! Keep up the good work :) `

## Screenshots (if appropriate):
r1
r2

### CODE REVIEW

**Comment by zekth** [comment]
May you edit and describe your PR?

**Comment by AymenRachid** [comment]
> May you edit and describe your PR?

Done :)

File: assets/translations/fr.csv [hunk 1]
```diff
@@ -77,91 +77,91 @@ configuration:settings:name_required,Nom requis
...
-edit,##
+ok,ok
+cancel,annuler
+documentation,documentation
+errors,erreurs
+since,depuis
+running,fonctionne
+health_chart:cpu_memory,diagramme de santé
+fatal_error:fatal_error,erreur fatale:erreur fatale
+fatal_error:connection_lost,connection perdue
```
Comment (zekth) [inline assets/translations/fr.csv]: connexion

**Comment by AymenRachid** [comment]
Edited the content of the pull request ! 
Thought of a mobile app along with the react front-end.

**Comment by ranjib** [comment]
Im closing this due to inactivity. Lets move the app code in separate repo. and have the translation in a separate pr.

**Pull request closed without merging**